wiki/Website-Markdown.md

```diff
@@ -191,6 +192,28 @@ Some text content here
 Some other content here
 ```
 
+#### Video
+
+Video assets are not natively supported in markdown, but you can easily insert a `.mp4` or similar file type into a markdown with a basic understanding of HTML:
+
+```md
+<video width="100%" controls loop>
+  <source
+    src="/assets/[components][patterns]/{name-of-component-or-pattern}/{path-to-video-file}.mp4"
+    type="video/mp4"
+  />
+</video>
+```
+
+This example uses the native HTML video tag with a couple of key attributes:
+
+- `controls` indicating that the video player should display play, pause, volume, and other basic controls.
```

Comment:
```suggestion
- `controls` indicating that the video player should display "play", "pause", "volume", and other basic controls.
```

Reply:
I added the quotes around the names of the control types, otherwise it seems harder to parse to me. (I had to read through it twice to understand that those were names of controls and not verbs in the sentence.)

Btw, for something representing actual code, it makes sense to use backticks to mark, it but for file names and names of file types (Ex: "mp4"), they aren't actually code, so I would use regular quotes for those. (They are just strings or names.)